libraries/Wire/src/Wire.h

```diff
@@ -28,23 +28,27 @@ extern "C" {
 #include "utility/twi.h"
 }
 
+// Minimal buffer length. Buffers length will be increased when needed,
+// but TX buffer is limited to a maximum to avoid too much stack consumption
 #define BUFFER_LENGTH 32
+#if !defined(WIRE_MAX_TX_BUFF_LENGTH)
```

Comment:
It should probably be fine to specify that it is limited to uint16_t size ?

Reply:
Done